cmd/guaccollect/cmd/files.go

```diff
@@ -171,7 +171,9 @@ func initializeNATsandCollector(ctx context.Context, pubsubAddr string, blobAddr
 		if err := collector.Collect(ctx, emit, errHandler); err != nil {
 			logger.Errorf("Unhandled error in the collector: %s", err)
 		}
-		done <- true
+		if !isPoll { // poll collectors should not exit
+			done <- true
+		}
```

Comment:
The oci and deps dev collector already support polling, this isn't necessary. A polling collector should exit when the context is cancelled.

Reply:
Removed this, thanks for the review!